.github/workflows/pytest.yml

```diff
@@ -15,8 +15,7 @@ env:
 
 jobs:
   build:
-    runs-on: ubuntu-18.04
-    container: python:3.8
+    runs-on: ubuntu-latest
```

Comment:
please use exact version name to avoid issues when new ubuntu arrives

Reply:
I'm against pinning this to a particular version as any version we put here will remain here for years.

Reply:
Wouldnt be a better solution to do a package upgrade chore task every other month? This would address python deps too.

Reply:
We bump python to 3.9 a long time ago. In this CI config, I change python from 3.8 to 3.9 this week when CI stop working.